tests/integration/test_firehose.py

```diff
@@ -429,9 +429,9 @@ def test_delivery_stream_with_kinesis_as_source(
                 },
             },
         )
-
-        assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
-
-        firehose_client.delete_delivery_stream(DeliveryStreamName=delivery_stream_name)
-        kinesis_client.delete_stream(StreamName=stream_name)
-        s3_client.delete_bucket(Bucket=s3_bucket)
+        try:
```

Comment:
I think if the evaluation gets this far, the resources cannot really be there anymore (since the client delete functions would raise an error).
Maybe we could use fixtures here which perform the cleanup properly after the test finishes?

Reply:
Could also use the cleanups fixture here which provides a fairly easy migration path

`cleanups.append(lambda: firehose_client.delete_delivery_stream(DeliveryStreamName=delivery_stream_name))`